### PR TITLE
Account for predefined set names

### PR DESCRIPTION
**Pull request by thpani**
Predefined set `BOOLEAN` requires special conversion of its Quint name `Bool`.

- [x] Tests added for any new code
- [x] Ran `make fmt-fix` (or had formatting run automatically on all files edited)
- [ ] ~Documentation added for any new functionality~
- [ ] ~[Entries added to `./unreleased/`][changelog format] for any new functionality~

[changelog format]: https://github.com/informalsystems/apalache/blob/main/CONTRIBUTING.md#how-to-record-a-change

### CODE REVIEW

File: tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala [hunk 1]
```diff
@@ -122,6 +122,9 @@ class TestQuintEx extends AnyFunSuite {
 
     // Names and parameters
     val name = e(QuintName(uid, "n"), QuintIntT())
+    val nameBoolSet = e(QuintName(uid, "Bool"), QuintSetT(QuintBoolT()))
+    val nameIntSet = e(QuintName(uid, "Int"), QuintSetT(QuintIntT()))
+    val nameNatSet = e(QuintName(uid, "Nat"), QuintSetT(QuintIntT()))
```
Comment (shonfeder) [inline tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala:127]: If you don't plan to reuse the named setr, you could create then inline in the test, just FYI. But this also looks great.

**Review by shonfeder — APPROVED** [review]
Looks great! Good catch :)

File: tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala (hunk 1 above)
Comment (thpani) [reply tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala:127]: Oh, just found `nam`.

Comment (thpani) [reply tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala]: 1c06d9d70

**Pull request merged**